### DISCUSSION TITLE
Get index in optionmenu

### DISCUSSION THREAD

**Discussion by B16f00t** [TomSchimansky/CustomTkinter#1004] [Q&A] [answered]
I have the following OptionMenu:

opt_lang = ["spa", "eng, "fre"]
 self.language_mode_menu = customtkinter.CTkOptionMenu(self.login_frame, values=opt_lang,                                                       command=self.change_language)

I need to get the index when I select an option, not the text.

**Comment by B16f00t** [comment, marked as answer]
I solved it.

    def change_language(self, language):
        n = 0
        for i in opt_lang:
            if i == language:
                print(n)
            n +=1

**Reply by TomSchimansky** [reply]
You can write it simpler with:
```python
def change_language(self, language):
    print(opt_lang.index(language))
```
